package.json

```diff
@@ -34,6 +34,7 @@
     "@openedx/frontend-plugin-framework": "^1.7.0",
     "@openedx/paragon": "^23.4.5",
     "@tanstack/react-query": "^5.83.0",
+    "camelcase-keys": "^9.1.3",
```

Comment:
we don't need a new library for this, frontend-platform has a utility for this purpose https://openedx.github.io/frontend-platform/module-Utilities.html#~camelCaseObject